R/applications.R

```diff
@@ -39,41 +43,79 @@ applications <- function(account = NULL, server = NULL) {
   serverDetails <- serverInfo(accountDetails$server)
   client <- clientForAccount(accountDetails)
 
-  # retreive applications
+  isConnect <- !isShinyapps(accountDetails)
```

Comment:
Note that there's also a third kind of account (Rpubs) but you don't need to worry about it here.